code/modules/clothing/chameleon.dm

```diff
@@ -72,7 +72,8 @@
 	name = "Select Chameleon Outfit"
 	button_icon_state = "chameleon_outfit"
 	var/list/outfit_options //By default, this list is shared between all instances. It is not static because if it were, subtypes would not be able to have their own. If you ever want to edit it, copy it first.
-
+	syndicate = TRUE
```

Comment:
Doesnt this mean only syndicates can use the outfit selector despite there being a non syndicate variant?